README.md

```diff
@@ -76,20 +86,43 @@ because there are moments where we're expecting certain things to
 happen (e.g., merge conflicts) and we want everyone to be in roughly
 the same place when that happens.
 
-This definitely will _not_ provide a comprehensive overview of the
-_many_ features `git` provides. There are a lot of on-line resources
-that can provide additional information, including:
-
+### Resources for learning `git` and friends
+
+This lab experience definitely will _not_ provide a comprehensive overview of the
+_many_ features `git` provides, and there are a _lot_ of on-line resources
+that can provide additional information.
+
+Some that you might want to look at right away
+to help you get up to speed on `git` and its
+friends include:
+
+* A few of the _many_ GitHub tutorials and
+  docs. There are a _lot_ more than just these, but these are short and pretty easy to spin through, and cover the key concepts we’ll need at the beginning of the semester.
+  * [GitHub Hello World](https://guides.github.com/activities/hello-world/) (a really nice,
+    short introduction to the key ideas)
+  * [Getting started with GitHub](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github)
+  * [Understanding the GitHub flow](https://guides.github.com/introduction/flow/)
+  * [Mastering Markdown](https://guides.github.com/features/mastering-markdown/)
+    (Markdown is the markup language used
+    by Slack, GitHub, and _many_ other
+    online tools, so understanding at least
+    the basics is really valuable.)
```

Comment:
Should this be on a sub-bullet like the one below is?

Reply:
Yup, it almost certainly should. Thanks!